docs/product/dev-toolbar/setup.mdx

```diff
@@ -9,50 +9,58 @@ description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry in
   [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
 </Alert>
 
-## Set Up Your Web App
+## Pre-requisites
 
-<Alert level="warning">
-    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite to using the Dev Toolbar. Tracing is used to collect page-specific issues and feedback through transactions.
-</Alert>
+For the Sentry toolbar to work best [enable tracing](/platforms/javascript/tracing/) in your app. With tracing enabled the Dev Toolbar will be able to associate issues and feedback with the current url in the browser location.
 
-You need to complete two steps to get the toolbar rendered on the page:
-1. Add or dynamically inject the following script into your web app:
-    ```html
-    <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
-    ```
-    It is recommended that you add the script tag at the bottom of the page so it doesn’t block other critical JavaScript.
-
-2. Call `window.SentryToolbar.init(initConfig)` to set up a toolbar instance on each page where you want to see the Dev Toolbar. Take care to also [conditionally load the toolbar in your production environment](#deploying-to-production-environments).
-    ```html
-    <html>
-    <head>...</head>
-    <body>
-        ...
-        <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
-        <script>
-        window.SentryToolbar.init({ ... });
-        </script>
-    </body>
-    </html>
-    ```
-
-3. Edit [Project Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/projects/) in Sentry so the Toolbar is allowed to connect from your dev, staging, and production domains.
-  ![Sentry's Dev Toolbar Settings Page](./img/sentry-project-settings-toolbar.png)
+## Install
 
+The first step is to include the toolbar code in your app.
```

Comment:
```suggestion
The first step is to include the Toolbar code in your app.
```

Reply:
I won't edit the rest of the 'toolbar' mentions -- I'll let you decide if you want to keep the capitalization 😅 I think it makes sense since this is referring to our product Dev Toolbar and not a generic toolbar (say, the one in screenshot annotaitons)